google/cloud/commerce/README.md

```diff
@@ -0,0 +1,57 @@
+# Cloud Commerce Consumer Procurement API C++ Client Library
+
+This directory contains an idiomatic C++ client library for the
+[Cloud Commerce Consumer Procurement API][cloud-service-docs], a service that enables consumers to procure products served by Cloud Marketplace platform.
```

Comment:
We normally reformat these to stay within 80 columns.

Reply:
https://github.com/googleapis/google-cloud-cpp/pull/12252 addresses this